stock_vertical_lift/static/src/js/vertical_lift.js

```diff
@@ -44,6 +45,53 @@ var ExitButton = FieldInteger.extend({
             clear_breadcrumbs: true,
         });
     },
+
+});
+
+
+FormController.include({
```

Comment:
Note: I tried to use a widget to handle the notifications handling and insert it in the view (rather than using the `FormController` for this), but on `this.reload()`, the 'start' method of the new widget was called before the 'destroy' of the previous one, so the channel was de-registered of the `bus_service`.